src/form/form-submit.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+class FormSubmit extends React.Component {
+  constructor(props) {
```

Comment:
Traditional constructors aren't needed here and if you use arrow functions for your methods, it will bind them to this automatically. A more concise way to write this would be like:

```
state = {
    ...
}

handleChange = (event) => { ... }
```